### PR TITLE
Fix#1471 Fix `VaSelect` and `VaInput`(`textarea`) readonly

### LINKED ISSUE #1471: readonly does not works for va-input(textarea) / va-select
**Vuestic-ui version:** 1.3.4

**Steps to reproduce**
image

[https://codesandbox.io/s/peaceful-panna-xm5y0?file=/src/App.vue](https://codesandbox.io/s/peaceful-panna-xm5y0?file=/src/App.vue)
**What is the expected behavior?**

**What is the current behavior?**

**Other information**

### PR DESCRIPTION
**Pull request by RVitaly1978**
## Description
close #1471 

changes:
- [x] fix readonly in `va-select`
- [x] fix readonly in `va-input` `type=textarea`
- [x] update examples in docs and demo

chrome-capture
chrome-capture

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Review by LighthouseKeeperYN — APPROVED** [review]

File: packages/ui/src/components/va-select/VaSelect.vue [hunk 1]
```diff
@@ -498,18 +489,13 @@ export default defineComponent({
...
+      if (props.disabled || props.readonly) { return }
+      showDropdown()
+    }
 
     const onSelectClick = () => {
-      if (props.disabled) {
-        return
-      }
-
+      if (props.disabled || props.readonly) { return }
```
Comment (m0ksem) [inline packages/ui/src/components/va-select/VaSelect.vue:498]: Let's choose one of this readonly behaviors: 

Hide arrow icon. 😐
image

Show select options, but don't allow user to click on any options. 👍
image

File: packages/ui/src/components/va-select/VaSelect.vue [hunk 2]
```diff
@@ -498,18 +489,13 @@ export default defineComponent({
...
     }
 
-    // Focus and keyboard navigation
-
-    const isFocusedComputed = computed(() => {
-      // If we show dropdown content that means select is focused
-      return isFocused.value || showDropdownContent.value
-    })
+    const openDropdown = () => {
+      if (props.disabled || props.readonly) { return }
```
Comment (m0ksem) [inline packages/ui/src/components/va-select/VaSelect.vue:493]: This condition can be moved to `showDropdown` method.

File: packages/ui/src/components/va-select/VaSelect.vue [hunk 3]
```diff
@@ -664,6 +649,11 @@ export default defineComponent({
       clearIconProps,
     }
   },
+  // we will use this while we have problem with 'withConfigTransport'
```
Comment (m0ksem) [inline packages/ui/src/components/va-select/VaSelect.vue:652]: Isn't there the way to implement focus and blur?

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/va-select/VaSelect.vue (hunk 1 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-select/VaSelect.vue:498]: After discussion, we accepted the first option (with the icon color changed to inactive)

File: packages/ui/src/components/va-select/VaSelect.vue (hunk 2 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-select/VaSelect.vue:493]: fixed it

File: packages/ui/src/components/va-select/VaSelect.vue (hunk 3 above)
Comment (RVitaly1978) [reply packages/ui/src/components/va-select/VaSelect.vue:652]: done it

**Pull request merged**